### DISCUSSION TITLE
Trying to update from provider 78.* to 91.* and getting the message "addon not verified by Firefox"

### DISCUSSION THREAD

**Discussion by maxweisenfeld** [kewisch/gdata-provider#302] [Support] [answered]
Trying to update from provider 78.* to 91.* and getting the message "addon not verified by Firefox"

What am I doing wrong, please?

**Comment by kewisch** [comment, marked as answer]
You will need to download the add-on instead of installing it into Firefox. You can do so using right click and "save as".
